src/v2/Utils/getSaleOrLotTimerInfo.ts

```diff
@@ -0,0 +1,91 @@
+export interface Time {
+  days: string
+  hours: string
+  minutes: string
+  seconds: string
+}
+
+interface TimerInfo {
+  copy: string
+  color: string
+}
+
+// If `extendedBiddingEndAt` is passed in and non-null, the `time`
+// parameter refers to the time left in the extended period.
+export const getSaleOrLotTimerInfo = (
```

Comment:
I would change the signature of this method and pass these in via a `params` or `options` object. Right now it's a list of things, which includes a mix of optional and not-optional arguments w/ defaults. That may make it a bit easier to use/pass in values (plus I think there's a related spec failure).

Reply:
Good call! This is getting confusing. ALSO it's gonna be so nice when all we have to do is make these exact super small changes to the eigen version for popcorn.